querydsl-sql/src/main/java/com/querydsl/sql/SQLListeners.java

```diff
@@ -245,4 +245,9 @@ public void exception(final SQLListenerContext context) {
             listener.exception(context);
         }
     }
+
+    public Set<SQLDetailedListener> getListeners() {
```

Comment:
Okay, since we only iterate over it to copy it, so `Iterable` would work, but if you need to add anything `Collection` would work as well.
Keeping it `Set` is also an option, I was just wondering though.